apps/cron/src/tasks/processGemsPayout/deployTopReferrerRewardsContract.ts

```diff
@@ -0,0 +1,92 @@
+import { log } from '@charmverse/core/log';
+import { prisma } from '@charmverse/core/prisma-client';
+import { getCurrentSeasonStart, getCurrentSeasonWeekNumber, getLastWeek } from '@packages/dates/utils';
+import { getTopConnectorOfTheDay } from '@packages/scoutgame/topConnector/getTopConnectors';
+import { DateTime } from 'luxon';
+import { parseUnits } from 'viem';
+import { optimismSepolia } from 'viem/chains';
+
+import { createSablierAirdropContract } from './createSablierAirdropContract';
+import { optimismTokenDecimals, optimismTokenAddress } from './deployNewScoutRewardsContract';
+
+const REFERRAL_CHAMPION_REWARD_AMOUNT = parseUnits('25', optimismTokenDecimals).toString();
+
+export async function deployTopReferrerRewardsContract() {
```

Comment:
if only one person is receiving reward, send something small to NULL address (0x000....)